ci/test/00_setup_env_native_previous_releases.sh

```diff
@@ -20,8 +20,8 @@ export BITCOIN_CONFIG="\
  -DWITH_ZMQ=ON -DBUILD_GUI=ON -DREDUCE_EXPORTS=ON \
  -DCMAKE_BUILD_TYPE=Debug \
  -DCMAKE_C_FLAGS='-funsigned-char' \
- -DCMAKE_C_FLAGS_DEBUG='-g0 -O2' \
+ -DCMAKE_C_FLAGS_DEBUG='-g2 -O2' \
```

Comment:
Any reason to set this to `-g2`, rather than `-g3` (the default for the Debug build type)?.